app/Console/Kernel.php

```diff
@@ -21,7 +24,11 @@ protected function schedule(Schedule $schedule): void
 
         $schedule->command(DispatchQueueCheckJobsCommand::class)->everyMinute();
 
-        //should be the last command per: https://spatie.be/docs/laravel-health/v1/available-checks/schedule
+        $schedule->command(PruneCommand::class, [
+            '--model' => [Audit::class, AssistantChatMessageLog::class],
+        ]);
```

Comment:
Can this include the setting that were on the one we deleted?

```
->daily()
->evenInMaintenanceMode()
->onOneServer();

```

Reply:
Done